DEVELOPING.md

```diff
@@ -4,6 +4,21 @@
 -   `npm run build` to compile cypher builder library
 -   `npm run docs` to generate the API reference docs
 
+## Link Cypher Builder locally with yarn
+
+In the Cypher Builder folder run:
+
+-   `yarn link`
+-   `yarn build`
+
+In the root of the package run:
+
+-   `yarn link -p [path-to-local-cypher-builder]
+
+To unlink, in the graphql project:
```

Comment:
Actually, just a thought, should this have the word `graphql` in it?

Reply:
nope, my bad I removed all references and I missed this one

```suggestion
To unlink, in the project using cypher-builder:
```